composer.json

```diff
@@ -13,28 +16,28 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "roave/security-advisories": "dev-master",
+        "zendframework/zend-component-installer": "^0.7.0",
```

Comment:
That depends on what the roadmap is. If 1.0 is out anytime soon without BC breaks, it's better to keep 1.0 so everyone who installs expressive with the skeleton will get the stable version.